scripts/homebrew/install_build_dependencies.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+brew install libheif dlib jpeg
```

Comment:
`brew update` installs the latest versions by default. I don't think it's a good practice to install other packages that are not related to the repo.

`apt` version to do `apt update` is because it also works in the container, which requires `apt update`.

Reply:
Most probably, you mixed the `update` with the `upgrade`:

Both `apt` and `brew` run `update` to refresh their local repos (similar to `git fetch`). Brew also updates itself, but this aspect can be safely ignored here.

To install newer versions of already installed packages, both managers need to run the `upgrade` command

Reply:
Yes. You are right. Done.